ff_format.c

```diff
@@ -252,13 +252,13 @@ static FF_Error_t prvFormatGetClusterSize( struct xFormatSet * pxSet,
 
     for( ; ; )
     {
-        int32_t groupSize;
+        uint32_t groupSize;
```

Comment:
Change it to `ulGroupSize`?

Reply:
feel free to change once this has been updated.  I truly don't see the point in adding superfluous prefixes to values to identify the type when the IDE's these days can tell you that with pop-ups or other features.  My 2 cents but if that is the format or flow then I suggest you add a tool to check and fix those for you since it is a simple thing to do automatically.
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

I understand you use uncrustify for this process, but it seems as though the bots for doing that aren't working.